workspaces/index.md

```diff
@@ -20,11 +20,15 @@ icon:
 description: "Learn about development workspaces in Coder."
 ---
 
-A workspace is your remote development environment. All workspaces are based on
-a shared [image](../images/index.md) and therefore reproducible.
+A workspace is your remote development environment.
 
 ![Workspaces](../assets/workspaces/workspaces.svg)
 
+All workspaces are based on a shared [image](../images/index.md) and therefore
```

Comment:
```suggestion
Coder creates workspaces using a [shared container image](../images/index.md), which improves their reproducibility.
```

The phrase "reproducible" means different things to different people, so we should either elaborate on what we mean by reproducible here, or acknowledge that Coder increases reproducibility but cannot guarantee it (many subtle things make builds non-reproducible, and the [Reproducible Builds project](https://reproducible-builds.org/) describes more of the background)